plugin/metrics/prometheus/options.go

```diff
@@ -63,9 +81,29 @@ func NewOptions(primaryNamespace string) *Options {
 // AddFlags from this storage to the CLI.
 func (opt *Options) AddFlags(flagSet *flag.FlagSet) {
 	nsConfig := &opt.Primary
-	flagSet.String(nsConfig.namespace+suffixServerURL, defaultServerURL, "The Prometheus server's URL, must include the protocol scheme e.g. http://localhost:9090")
-	flagSet.Duration(nsConfig.namespace+suffixConnectTimeout, defaultConnectTimeout, "The period to wait for a connection to Prometheus when executing queries.")
-	flagSet.String(nsConfig.namespace+suffixTokenFilePath, defaultTokenFilePath, "The path to a file containing the bearer token which will be included when executing queries against the Prometheus API.")
+	flagSet.String(nsConfig.namespace+suffixServerURL, defaultServerURL,
```

Comment:
I considered adding a single "OTEL component type" (value one of: "processor" or "connector") flag which hardcodes all the breaking changes to the component type, instead of all these new flags, but there were some drawbacks that made me avoid this path:

- Couples Jaeger too closely to OTEL collector.
- If the spanmetrics connector decides to change a metric name, it would involve a code change in Jaeger. Having separate config options means users can just update their config in this scenario.

Let me know if you disagree.